src/workloads/execution/ShardedLookup.yml

```diff
@@ -0,0 +1,116 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+
```

Comment:
Could you add a Description field here to give a brief overview about what this workload will test?

Reply:
Done!